packages/backend/src/graphql/queries/get-flow.ts

```diff
@@ -1,6 +1,13 @@
 import type { QueryResolvers } from '../__generated__/types.generated'
 
+const UUID_REGEX = /^[a-z,0-9,-]{36,36}$/
+
 const getFlow: QueryResolvers['getFlow'] = async (_parent, params, context) => {
+  // To avoid the gibberish error code if a user keys in an invalid editor route e.g. editor/123
+  if (!UUID_REGEX.test(params.id)) {
+    throw new Error('The pipe id in your URL is of an invalid format.')
+  }
+
```

Comment:
nit: I would avoid regex if possible, plus UUIDv4 is actually stricter than this regex
```suggestion
  if (!z.string().uuid().safeParse(params.id).success) {
    throw new Error('Please provide a valid pipe ID.')
  }
```

Reply:
ok thanks!